ground/src/main/res/values/strings.xml

```diff
@@ -69,7 +69,7 @@
   <string name="date">Date</string>
   <string name="time">Time</string>
   <!-- Other strings (to be organized) -->
-  <string name="sync_status">History and sync status</string>
+  <string name="sync_status">History</string>
```

Comment:
The title in the nav menu should still be "History and sync status". @vittorino  Was the different labels between that and the status screen header intentional?